src/components/ValueInput/ValueInput.js

```diff
@@ -188,6 +189,7 @@ export const ValueInputComponent = ({
   };
 
   const handleUsePercent = async (percent: number) => {
+    Keyboard.dismiss();
```

Comment:
Pleased add `keyboardShouldPersistTaps` for SendContainer, as now when in Send screen, tapping max value (not percent amount) first cause keyboard dismiss but not applcation of max value, only second tap cause ValueInput to apply max value.